### ISSUE TITLE
Mobile viewport not set

### ISSUE THREAD

**Issue by jackkinsella** [jackkinsella/confessions#96]
https://www.google.com/webmasters/tools/mobile-friendly/?url=https://www.jackkinsella.ie/&utm_source=gws&utm_medium=metaline&utm_campaign=notmobilefriendy

**Comment by adamniedzielski** [comment]
Here is the commit which adds it https://github.com/jackkinsella/confessions/commit/4a170688ec22fae25ce26ddd7113fc8271d56cae but I made it in a separate branch - `mobile-viewport`.

The reason is that it's hard to decide whether the mobile version looks better or worse after this change, but the differences are huge. Certainly we need this viewport tag for the application to be responsive (it's a standard practice and I copy-pasted the code from Oxbridge Notes), but the CSS needs to be changed as well.

I recommend you to use Chrome DevTools and launch Device Mode to compare how the application looks before and after this change.

**Comment by jackkinsella** [comment]
it's a lot worse IMO, but we definitely need to make the switch to using the tag. I wish I put that tag in _before_ I started designing. oh well, lesson learned 

I'll work on fixing this over the coming week or two

**Comment by adamniedzielski** [comment]
Yeah, in fact I'm quite surprised that it's not part of the default Rails template.
